src/azul/service/manifest_service.py

```diff
@@ -425,27 +429,22 @@ def _dss_url(self, file):
     @cachedproperty
     def manifest_content_hash(self) -> int:
         es_search = self._create_request()
-        es_search.aggs.metric(
-            'hash',
-            'scripted_metric',
-            init_script='''
-                params._agg.fields = 0
-            ''',
-            map_script='''
-                for (bundle in params._source.bundles) {
-                    params._agg.fields += (bundle.uuid + bundle.version).hashCode()
-                }
-            ''',
-            combine_script='''
-                return params._agg.fields.hashCode()
-            ''',
-            reduce_script='''
-                int result = 0;
-                for (agg in params._aggs) {
-                    result += agg
-                }
-                return result
-          ''')
+        es_search.aggs.metric('hash',
```

Comment:
Why does this undo a change I had requested in a previous PR?

Reply:
It was a bad conflict resolution. I had fixed it and failed to catch it again on the rebase of the latest changes.